Tests/TensorFlowTests/OperatorTests/MathTests.swift

```diff
@@ -514,6 +521,7 @@ final class MathOperatorTests: XCTestCase {
         ("testIsFinite", testIsFinite),
         ("testIsInfinite", testIsInfinite),
         ("testIsNaN", testIsNaN),
+        ("testCosineDistance", testCosineDistance),
```

Comment:
Thanks @anigasan! Since you defined this under `testCosineSimilarity` above, please also put this line under the `testCosineSimilarity` line here.

Reply:
I have made the necessary changes needed